### PR TITLE
Update preparing_for_genesis.md

### PR DESCRIPTION
**Pull request by 2dpetkov**

### CODE REVIEW

**Comment by 2dpetkov** [comment]
PR is a draft, because we need the wethDYDX deployment address

File: pages/validators/preparing_for_genesis.md [hunk 1]
```diff
@@ -16,7 +16,44 @@ These are necessary steps to register your validator in the genesis and prepare
...
+
+Transform your `accAddress` to hexidecimal bytes
+```bash
+git clone https://github.com/dydxprotocol/v4-chain.git
+cd v4-chain/protocol
+go run scripts/bech32_to_hex/bech32_to_hex.go -address <bech32_address>
+```
+
+To initiate the migration of your self-delegation `DYDX` amount from Ethereum
+1. navigate to the [`bridge` function](https://etherscan.io/address/TODO) of `wethDYDX` on Etherscan
```
Comment (pswies) [inline pages/validators/preparing_for_genesis.md:42]: As above, TODO.

**Review by pswies — COMMENTED** [review]
Probably OK but added some observations.

File: pages/validators/preparing_for_genesis.md [hunk 2]
```diff
@@ -16,7 +16,44 @@ These are necessary steps to register your validator in the genesis and prepare
...
+
+To participate in the potential Genesis process, there is a requirement for genesis Validators to self-delegate `DYDX` tokens, which necessitates locking `DYDX` on Ethereum ([link 1](https://www.dydx.foundation/blog/exploring-the-future-of-dydx), [link 2](https://www.dydx.foundation/blog/update-on-exploring-the-future-of-dydx), [link 3](https://docs.dydx.community/dydx-token-migration/migration-of-dydx-from-ethereum-to-dydx-chain/migration-and-bridge-overview)).
+dYdX Operations subDAO suggest a minimum of `1000 DYDX`, although the amount required to self-delegate is flexible and is validators' choice.
+
+To approve the `wethDYDX` Smart Contract to transfer the bridging amount of `DYDX` on your behalf
+1. navigate to [`approve` function](https://etherscan.io/address/0x92D6C1e31e14520e676a687F0a93788B716BEff5#writeContract#F2) of the `DYDX` Token on Etherscan
+2. click `Connect to Web3`
+4. input [`wethDYDX` Smart Contract Address](https://etherscan.io/address/TODO) and desired amount, e.g. `1000000000000000000000` for bridging `1000 DYDX` tokens ([`DYDX` Token decimals](https://etherscan.io/address/0x92D6C1e31e14520e676a687F0a93788B716BEff5#readContract#F24) is `18`)
```
[the remark this answers is not in the document]
Comment (2dpetkov) [reply pages/validators/preparing_for_genesis.md]: Thanks, that's the wethDYDX address, should be updated now

File: pages/validators/preparing_for_genesis.md [hunk 3]
```diff
@@ -16,7 +16,44 @@ These are necessary steps to register your validator in the genesis and prepare
 
 ## Timeline
 
-See the [Mainnet Launch Schedule](https://v4-mainnet-docs.vercel.app/mainnet/schedule) for `dydx-mainnet-1`. Please make sure to submit your gentx by the deadline highlighted in the schedule.
+See the [Mainnet Launch Schedule](https://v4-mainnet-docs.vercel.app/mainnet/schedule) for `dydx-mainnet-1`. Please make sure to prepare your `DYDX` tokens for self-delegation, and submit your gentx by the deadlines highlighted in the schedule.
+
+## Preparing `DYDX` for self-delegation
+
+To participate in the potential Genesis process, there is a requirement for genesis Validators to self-delegate `DYDX` tokens, which necessitates locking `DYDX` on Ethereum ([link 1](https://www.dydx.foundation/blog/exploring-the-future-of-dydx), [link 2](https://www.dydx.foundation/blog/update-on-exploring-the-future-of-dydx), [link 3](https://docs.dydx.community/dydx-token-migration/migration-of-dydx-from-ethereum-to-dydx-chain/migration-and-bridge-overview)).
```
[the remark this answers is not in the document]
Comment (2dpetkov) [reply pages/validators/preparing_for_genesis.md]: It's not for dYdX Operations subDAO to take accountability for the launch and its success.

**Review by pswies — APPROVED** [review]
LGTM!

**Pull request merged**